src/gui/screen_lan_settings.c

```diff
@@ -253,19 +384,26 @@ static int screen_lan_settings_event(screen_t *screen, window_t *window,
         } else {
             if (_load_ini_file()) {
                 if (gui_msgbox("Settings successfully loaded", MSGBOX_BTN_OK | MSGBOX_ICO_INFO) == MSGBOX_RES_OK) {
+                    plsd->items[MI_TYPE].item.wi_switch_select.index = config.lan_flag & LAN_EEFLG_TYPE ? 1 : 0;
+                    window_invalidate(plsd->menu.win.id);
+                    if (!(config.lan_flag & LAN_EEFLG_TYPE)) {
+                        _refresh_addresses(screen);
+                        conn_flg = true;
+                    } else {
+                        _addrs_to_str(plsd->mac_addr_str, 0);
+                        plsd->text.text = plan_str;
+                        plsd->text.win.flg |= WINDOW_FLG_INVALID;
+                    }
                 }
-                _addrs_to_str(plsd->mac_addr_str, 0);
-                plsd->text.win.flg |= WINDOW_FLG_INVALID;
 
             } else {
-                if (gui_msgbox("File \"lan_settings.ini\" not found in the root directory of the USB flash disk.",
+                if (gui_msgbox("Addresses are not valid or the file \"lan_settings.ini\" is not in the root directory of the USB flash disk.",
```

Comment:
check proper wording with the content team

Reply:
In progress.

Reply:
no reply form content yet, will be updated as they send us the text